docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.icons.scss

```diff
@@ -0,0 +1,30 @@
+@mixin hb-external-link-icon {
+  margin-right: hb-calculate-rems(20px);
+
+  @include hb-colorful {
+    $hb-lighten-color--primary: lighten(hb-colorful-variation(primary), 10%);
+    $hb-color--primary: hb-colorful-variation(primary);
+    position: relative;
+
+    &::before {
+      content: '';
+      background: svg('<svg viewBox="0 0 12 12"><path d="M11.273 5.818v5.455H.727V.727h5.091m1.455 0h4v4m0-4L6.182 5.818" fill="transparent" stroke="#{$hb-lighten-color--primary}" stroke-width="2" stroke-miterlimit="10"/></svg>') center center no-repeat;
+      height: hb-calculate-rems(12px);
+      width: hb-calculate-rems(12px);
+      display: inline-block;
+      vertical-align: middle;
+      position: absolute;
+      top: hb-calculate-rems(4px);
+      right: hb-calculate-rems(-18px);
+    }
+
+    &:hover::before,
+    &:focus::before {
+      background: svg('<svg viewBox="0 0 12 12"><path d="M11.273 5.818v5.455H.727V.727h5.091m1.455 0h4v4m0-4L6.182 5.818" fill="transparent" stroke="#{$hb-color--primary}" stroke-width="2" stroke-miterlimit="10"/></svg>') center center no-repeat;
+    }
+  }
+}
+
+@mixin hb-link-arrow-icon($color: $hb-color--black) {
+  background: no-repeat left / 55% svg('<svg viewBox="0 0 23 24"><path fill="#{$color}" d="M11 23l-2-2 7-7H0v-3h16L9 3l2-2 10 10 1 1-11 11z"/></svg>');
```

Comment:
Yay! is there anything else that can be extracted here? or it's probably better to keep all the transforms on hover in another area?

Reply:
I almost went the route of creating a mixin that included the transform, hover, and focus states, but then I looked at how this sits inside of the `hb-colorful` theme and decided to keep the scope narrowed down to just the icon.

![Screen Shot 2020-03-10 at 10 35 28 AM](https://user-images.githubusercontent.com/12678977/76323361-ff024780-62ba-11ea-9c24-c79317c7953b.png)

Perhaps when we implement additional themes there will be an opportunity to refactor.

Reply:
ok sounds good!